### DISCUSSION TITLE
模块‘element-plus’中没有导出的成员‘xxx’, ts错误

### DISCUSSION THREAD

**Discussion by Sami-zzz** [element-plus/element-plus#12744] [Q&A]
我在vite+ts下仅使用按需自动导入element-plus，在书写以下import语句时都显示此问题
![](https://i.328888.xyz/2023/05/09/iQ9hjC.jpeg)
请求大家的帮助

**Comment by hec9527** [comment]
你应该是使用vite创建的项目吧，吧tsconfig.json 修改一下就行了


"moduleResolution": "Node",

**Reply by Sami-zzz** [reply]
谢谢，之前在issue下看到解决方法了，是这样
